app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -519,7 +509,6 @@ def dialog_partial_for_workflow
     case workflow
     when MiqProvisionVirtWorkflow                    then "shared/views/prov_dialog"
     when ManageIQ::Providers::Foreman::ConfigurationManager::ProvisionWorkflow then "prov_configured_system_foreman_dialog"
-    when MiqHostProvisionWorkflow                    then "prov_host_dialog"
```

Comment:
You should also be able to remove `./app/views/miq_request/_prov_host_dialog.html.haml`, right?